Sources/xcodeproj/Project/XcodeProj.swift

```diff
@@ -81,10 +91,18 @@ extension XcodeProj: Writable {
     ///   If false will throw error if project already exists at the given path.
     public func write(path: AbsolutePath, override: Bool = true, outputSettings: PBXOutputSettings) throws {
         try path.mkpath()
-        try writeWorkspace(path: path, override: override)
-        try writePBXProj(path: path, override: override, outputSettings: outputSettings)
-        try writeSchemes(path: path, override: override)
-        try writeBreakPoints(path: path, override: override)
+        try OSLogger.instance.log(category: String(describing: self), name: "Write workspace", path.asString) {
```

Comment:
You could get the category automatically using `#file`

Reply:
Good idea. Let me update it.